api/store.js

```diff
@@ -0,0 +1,176 @@
+const logger = require('./logger')('blob store');
+const fs = require('fs');
+const stream = require('stream');
+const fsBlobStore = require('fs-blob-store');
+
+const isReadableStream = obj => obj instanceof stream.Readable;
+
+// Creates a filestream blob store that reads/writes to
+// /dev/null.  Can't use the fsBlobStore for this because
+// it tries to create /dev/null and...  well, that doesn't
+// fails, as we might reasonably expect.
+const nullBlobStore = () => ({
+  createReadStream: () => fs.createReadStream('/dev/null'),
+  createWriteStream: () => fs.createWriteStream('/dev/null'),
```

Comment:
Creates a dependency on Linux-y systems for now.  There's almost certainly a better way to do this, but it gets us going for now.  Longterm, we might don't need this at all.  Once we've got access to S3, we could create a staging bucket and a production bucket, and bam, we don't have to just pretend like we're storing files.  😛